mtp_api/urls.py

```diff
@@ -46,3 +48,9 @@
 
     url(r'^$', lambda request: HttpResponse(content_type='text/plain', status=204)),
 ]
+if settings.ENVIRONMENT in ('test', 'local'):
```

Comment:
My reasoning for using an inclusive list rather than an exclusive list is just in case we end changing the name of the production cluster (can't think why this would happen, maybe a change from `prod` to `production`). But I can change it if you want.